manage/import-export-dropbox.js

```diff
@@ -0,0 +1,191 @@
+/* global messageBox */
+'use strict';
+
+const DROPBOX_API_KEY = 'uyfixgzre8v1bkg';
+const FILENAME_ZIP_FILE = 'stylus.json';
+const DROPBOX_FILE = 'stylus.zip';
+const API_ERROR_STATUS_FILE_NOT_FOUND = 409;
+const HTTP_STATUS_CANCEL = 499;
+
+function messageProgressBar(data) {
+  return messageBox({
+    title: `${data.title}`,
+    className: 'config-dialog',
+    contents: [
+      $create('p', data.text)
+    ],
+    buttons: [{
+      textContent: t('confirmClose'),
+      dataset: {cmd: 'close'},
+    }],
+  }).then(_ => {
+    document.body.style.minWidth = '';
+    document.body.style.minHeight = '';
+  });
+}
+
+function hasDropboxAccessToken() {
+  return chromeLocal.getValue('dropbox_access_token');
+}
+
+function requestDropboxAccessToken() {
+  const browserApi = typeof browser === 'undefined' ? chrome : browser;
+  const client = new Dropbox.Dropbox({clientId: DROPBOX_API_KEY});
+  const authUrl = client.getAuthenticationUrl(browserApi.identity.getRedirectURL());
+
+  return browserApi.identity.launchWebAuthFlow({url: authUrl, interactive: true})
+  .then(urlReturned => {
+    const params = new URLSearchParams(new URL(urlReturned).hash.replace('#', '?'));
```

Comment:
due to a bug in older versions of Chrome we simply strip the character e.g. `'#', ''`